.github/workflows/quality_check_pydanticv2.yml

```diff
@@ -64,12 +64,8 @@ jobs:
         run: poetry add "pydantic=^2.0.3"
       - name: Install dependencies
         run: make dev
-      - name: Formatting and Linting
-        run: make lint
-      - name: Static type checking
-        run: make mypy
       - name: Test with pytest
-        run: make test
+        run: make test-pydanticv2
       - name: Security baseline
```

Comment:
We can remove all of it and keep only tests for Pydantic v2, since the entire code will be checked either way by the other workflow ;)

Reply:
I ketp this just for AppSec, but make sense.. removing.